xchange-okcoin/src/main/java/com/xeiam/xchange/okcoin/service/polling/marketdata/OkCoinMarketDataServiceRaw.java

```diff
@@ -39,9 +40,9 @@ public OkCoinTickerResponse getFuturesTicker(CurrencyPair currencyPair, FuturesC
     return okCoin.getFuturesTicker(OkCoinAdapters.adaptSymbol(currencyPair), prompt.getName());
   }
 
-  public OkCoinDepth getDepth(CurrencyPair currencyPair) throws IOException {
+  public OkCoinDepth getDepth(CurrencyPair currencyPair, Integer size) throws IOException {
```

Comment:
This should be done differently. Instead of changing method signature, add another one with different signature. Otherwise you create incompatilibity.